src/internal/Observable.ts

```diff
@@ -70,6 +70,12 @@ export class Observable<T> implements Subscribable<T> {
   }
 
   subscribe(observer?: PartialObserver<T>): Subscription;
+  /** @deprecated */
+  subscribe(next: null, error: null, complete: () => void): Subscription;
```

Comment:
I think `undefined` would be the way to go. At the moment, it's not possible to call `subscribe` and pass `null` as an argument without TypeScript complaining, so unless the existing signatures are altered to accept `null` arguments, I the deprecations will be unnecessary.

Reply:
There'd be no harm in using `null | undefined`, though.